### PR TITLE
feat(cli): add bun support

### PR DESCRIPTION
**Pull request by frankcalise**
## Please verify the following:

- [ ] `yarn test` **jest** tests pass with new tests, if relevant
- [ ] `README.md` has been updated with your changes, if relevant

## Describe your PR

- Adds support for the `bun` package manager 🎉 

image

### CODE REVIEW

**Review by morganick — APPROVED** [review]
Works as expected. I wonder if the commands are similar enough in all situations to where we could extract out a mapping of the commands for the package manager so the functions can be generic and the mapping can be updated per packager more easily.

**Comment by morganick** [comment]
Appears that tests are failing for it.

**Comment by jamonholmgren** [comment]
This PR for Gluegun will add better Bun support over there as well.

https://github.com/infinitered/gluegun/pull/783

**Comment by jamonholmgren** [comment]
IMO, we should consider moving to Bun for local development of Ignite as well as CI (like I'm doing with Gluegun CLI). It'll speed up tests a _lot_.

**Comment by frankcalise** [comment]
@jamonholmgren that looks like maybe you have `ios/` and `android/` dirs local left over from Ignite 8 dev. This branch is targeting Ignite 9 (although if we want to cherry pick it over we could)

**Review by jamonholmgren — APPROVED** [review]
Let's merge it!

File: PizzaBunApp
Comment (joshuayoes) [inline PizzaBunApp:1]: What is this git submodule here?

Comment (frankcalise) [reply PizzaBunApp:1]: Uh no idea, shouldn't be there. Good catch

Comment (frankcalise) [reply PizzaBunApp:1]: Should be gone now

**Pull request merged**